<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Makhtar</author>
 <class>QirdaWindow</class>
 <widget class="QMainWindow" name="QirdaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>453</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qirda - Infrared data exchanger</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/qirdaLogo-2.png</normaloff>img/qirdaLogo-2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelDiagnos">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="debuggingBox">
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="rcvBt">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Receive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="verbBt">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <underline>true</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="text">
             <string>Verbose</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearDiagnosticsBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Cl&amp;ear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelFiles2Send">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="inFilesBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>13</pointsize>
            <stylestrategy>PreferDefault</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SUSE Serif'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="addFileBt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Add Files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendBt">
            <property name="text">
             <string>&amp;Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearBt">
            <property name="text">
             <string>C&amp;lear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>&amp;Info</string>
    </property>
    <addaction name="actAbout"/>
    <addaction name="actHelp"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>&amp;Control</string>
    </property>
    <widget class="QMenu" name="menuLang">
     <property name="title">
      <string>&amp;Language</string>
     </property>
     <addaction name="actEnglish"/>
     <addaction name="actFrench"/>
    </widget>
    <widget class="QMenu" name="menuSysTrayIcon">
     <property name="title">
      <string>System Tray &amp;Icon</string>
     </property>
     <addaction name="actEnableTrayIcon"/>
     <addaction name="actDisableTrayIcon"/>
    </widget>
    <addaction name="actSelectRcvDir"/>
    <addaction name="menuLang"/>
    <addaction name="separator"/>
    <addaction name="menuSysTrayIcon"/>
    <addaction name="actDetect"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <addaction name="menuControl"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statBar"/>
  <action name="actAbout">
   <property name="text">
    <string>A&amp;bout Qirda</string>
   </property>
  </action>
  <action name="actSelectRcvDir">
   <property name="text">
    <string>Reception &amp;Directory...</string>
   </property>
   <property name="iconText">
    <string>Reception Directory</string>
   </property>
   <property name="toolTip">
    <string>Reception Directory</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actDetect">
   <property name="text">
    <string>D&amp;etect devices...</string>
   </property>
  </action>
  <action name="actEnglish">
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actFrench">
   <property name="text">
    <string>&amp;Francais</string>
   </property>
   <property name="iconText">
    <string>Francais</string>
   </property>
   <property name="toolTip">
    <string>Français</string>
   </property>
  </action>
  <action name="actHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actEnableTrayIcon">
   <property name="text">
    <string>&amp;Enable</string>
   </property>
  </action>
  <action name="actDisableTrayIcon">
   <property name="text">
    <string>Disa&amp;ble</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearDiagnosticsBt</sender>
   <signal>clicked()</signal>
   <receiver>debuggingBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
